### PR TITLE
Remove the dependency of meta master

### PR DESCRIPTION
**Pull request by yyongycy**
### What changes are proposed in this pull request?

Remove the dependency of meta master

### Why are the changes needed?

So far the worker and client are depending on the master for configuration, this is to remove the dependency.

### Does this PR introduce any user facing changes?
NA

### CODE REVIEW

**Comment by yyongycy** [comment]
@apc999  Please help take a look.

File: dora/core/server/worker/src/main/java/alluxio/worker/AlluxioWorker.java [hunk 1]
```diff
@@ -53,18 +53,6 @@ public static void main(String[] args) {
...
-    try {
-      RetryUtils.retry("load cluster default configuration with master", () -> {
-        InetSocketAddress masterAddress = masterInquireClient.getPrimaryRpcAddress();
-        Configuration.loadClusterDefaults(masterAddress, Scope.WORKER);
-      }, RetryUtils.defaultWorkerMasterClientRetry());
-    } catch (IOException e) {
-      ProcessUtils.fatalError(LOG,
-          "Failed to load cluster default configuration for worker. Please make sure that Alluxio "
-              + "master is running: %s", e.toString());
-    }
```
Comment (apc999) [inline dora/core/server/worker/src/main/java/alluxio/worker/AlluxioWorker.java:67]: also remove similar code  in `AlluxioJobWorker.java`?

Comment (apc999) [reply dora/core/server/worker/src/main/java/alluxio/worker/AlluxioWorker.java:67]: and `alluxio.conf.Configuration#loadClusterDefaults` as well as their references?

**Review by apc999 — APPROVED** [review]
thanks for the removal!

**Comment by yyongycy** [comment]
alluxio-bot, merge this please

**Pull request merged**